indico/htdocs/js/indico/widgets/person_link_widget.js

```diff
@@ -206,6 +212,9 @@
             if (person.isSubmitter === undefined) {
                 person.isSubmitter = options.defaults.isSubmitter;
             }
+            if (!person.id) {
```

Comment:
what if a person has id 0 (you can have a User with ID if you have a migrated DB)?